tests/ResourcePathResolveTest.php

```diff
@@ -24,54 +25,54 @@ public function setUp(): void
     public function test_it_can_resolve_api_changes()
     {
         $this->apiResourceManager->setVersion('1');
-        $classname = $this->callMethod($this->apiResourceManager, 'parseClassname', ['App\User']);
+        $classname = $this->callMethod($this->apiResourceManager, 'parseClassname', ['App\User', '1']);
         $this->assertEquals('\\App\\Http\\Resources\\App\\v1\\User', $classname);
 
         $this->apiResourceManager->setVersion('2');
-        $classname = $this->callMethod($this->apiResourceManager, 'parseClassname', ['App\Users']);
+        $classname = $this->callMethod($this->apiResourceManager, 'parseClassname', ['App\Users', '2']);
         $this->assertEquals('\\App\\Http\\Resources\\App\\v2\\Users', $classname);
 
         $this->apiResourceManager->setVersion('3');
-        $classname = $this->callMethod($this->apiResourceManager, 'parseClassname', ['App\User\Single']);
+        $classname = $this->callMethod($this->apiResourceManager, 'parseClassname', ['App\User\Single', '3']);
```

Comment:
It sounds like we need some data providers in this file.

Reply:
Yeah, we will delete this once Mazen finishes his refactor, to be honest

Reply:
This will be obsolete test yes